### ISSUE TITLE
Database throws exception on playercreation with valid inputs

### ISSUE THREAD

**Issue by Santaaa348** [bendixsonnenberg/CS-Select#143]
Instead it says "Email is already in use." (Organiser still works as expected)

**Comment by bendixsonnenberg** [comment]
stacktrace for 
```
java.lang.IllegalArgumentException: Email is already in use!
	com.csselect.database.mysql.MysqlUserAdapter.setEmail(MysqlUserAdapter.java:53)        com.csselect.database.mysql.MysqlDatabaseAdapter.createPlayer(MysqlDatabaseAdapter.java:269)
	com.csselect.user.management.PlayerManagement.register(PlayerManagement.java:24)
	com.csselect.API.APIFacadePlayer.register(APIFacadePlayer.java:39)
	com.csselect.API.httpAPI.Login.register(Login.java:56)
	com.csselect.API.httpAPI.Login.post(Login.java:38)
	com.csselect.API.httpAPI.Servlet.doPost(Servlet.java:102)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:660)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
```

**Comment by bendixsonnenberg** [comment]
probably caused by editing the database adapter to notifiy the user of an duplicate email address

**Comment by bendixsonnenberg** [comment]
``` java
        try {
            adapter = new MysqlPlayerAdapter(username, email, hash, salt);
        } catch (SQLException e) {
            e.printStackTrace();
            return null;
        }
        adapter.setEmail(email);
        adapter.setPassword(hash, salt);
        adapter.setUsername(username);
        return new Player(adapter);
```
this seems to set everything twice and causes the error

**Comment by SeineEloquenz** [comment]
Fixed via 16fd36931f22bdfca4e3b860395222a5493a2b91
